docs/src/userguide/memoryviews.rst

```diff
@@ -660,6 +660,18 @@ object handling. For the details of how to compile and
 call functions in C files, see :ref:`using_c_libraries`.
 
 
+Performance: Disabling initialization checks
+============================================
+
+When memoryviews are used as class attributes, initialization checks are performed
+by default. Those checks are executed everytime a memoryview is accessed, introducing
```

Comment:
I think the phase "initialization checks" sounds a bit like an internal detail - if you don't already know what it means then it isn't that obvious.

How about something spelling it out a little more, like "every time the memoryview is accessed, Cython adds a check to make sure that it has been initialized."?

I know what uses most of the same words, but it feels a bit clearer to me.

Thanks for the PR - I think it's useful to point it out in the relevant place in the documentation.

Reply:
> I think the phase "initialization checks" sounds a bit like an internal detail - if you don't already know what it means then it isn't that obvious.

That's right, users do not know this and need not to.
> How about something spelling it out a little more, like "every time the memoryview is accessed, Cython adds a check to make sure that it has been initialized."? I know what uses most of the same words, but it feels a bit clearer to me.

Yes, definitely; 0096d00 is a rewording attempt in this regards.